openlibrary/templates/books/edition-sort.html

```diff
@@ -26,6 +26,7 @@
     $ isbn_10 = book.isbn_10[0].replace("-", "")
 $elif isbn_13:
     $ isbn_10 = isbn_13_to_isbn_10(isbn_13)
+$ asin = isbn_10 or None
```

Comment:
We should check the asin field itself first. I'm not sure what type `book` is, but according to https://openlibrary.org/books/OL25426332M.json, maybe `(book.identifiers.amazon[0])`?

Reply:
punting for this PR, will consider for future ticket